### PR TITLE
[Expert] Mastery Shard Tweaks

### PR DESCRIPTION
**Pull request by MuteTiefling**
Mastery Shard tweaks in response to #5048

### CODE REVIEW

**Comment by MuteTiefling** [comment]
Replace Twisted Saplings in Nature's Aura Tree Ritual with various other Quark saplings, as they appear to be the only ones that are reliable...

**Comment by MuteTiefling** [comment]
Automation Mastery Shard

Re-balance of Automation Mastery. Increase to netherite crafters, includes a few other automation items. Still isn't "hard" per se, but perhaps a little more interesting and gets the player ready for what's to come. 
image
Moves some items into DIY kits to free up room for a few other items. 

Router Kit
image

Drone Kit
image

**Comment by MuteTiefling** [comment]
Aura Mastery Shard

Expanding the requirements a bit here. 
image


Now needs an automated packing setup for fireworks: 
image

Added in a shooting mark craft, and a box of memories for that:
image

Added a natural altar requirement as well. 
image

**Comment by MuteTiefling** [comment]
Culinary Mastery Shard
Add a new meal type!
image

Similar to the Student Meals, this is a sequential craft that will require automating a few different things, including more cooking pot and getting 'plates' of meals from the block form. 
image

JEI Descriptions added to help people with this: 
image

Also adds some much needed JEI Descriptions to the simplfarming brewing process:
image
image

**Comment by MuteTiefling** [comment]
Engineering Mastery Shard

Rework the main shard to put the Furnace Engine stuff in a box
image

Increase the difficulty slightly by including Resistive Heater instead
image

**Comment by MuteTiefling** [comment]
Excavation Mastery Shard

I think there was actually a transposing error on my part here since the shard needs 4 quarry boxes and the boxes each needed 4 sets of quarry items. Also reduces to one kit per shard instead of 2 per shard. 

image
image

**Comment by MuteTiefling** [comment]
Source Costs are reduced generally across the board. Some truly one-off crafts remain high, but most other crafts come down significantly. Having automated source is still a requirement, but for long crafting sessions, source transfer rate should no longer be the bottleneck.

**Comment by MuteTiefling** [comment]
Astronomy Mastery Shard

Update to Auto Iridescent Altar
image

The multiblock now contains attuned crystals for all possible constellations
image
image

This means it can now handle crafts that normally require a specific crystal, such as Irradiant Stars
image

**Comment by MuteTiefling** [comment]
Dimensional Mastery
Swap Disks for storage blocks, reduce count yet again
Remove disk drive entirely
Move teleportation stuff from Battle to Dimensional. Was original in Battle as it seemed to make sense that thes patrolling meka-suite wearers would need to get around, but I agree it fits better here. 
image

**Comment by MuteTiefling** [comment]
Botanical
Simple rebalance of existing materials with the minor addition of adding recessive sparks. 
Two shards per craft now, so it's down to 100 mana collectors from 400. Enough, I think, that you'll still want to automate it, but not insane. 
image

**Comment by MuteTiefling** [comment]
So this leaves Astronomy which might need a little more love, and Alchemy which needs a lot of love.

I'm short on ideas for both. Could use some input there.

**Comment by CaptainAlkohol** [comment]
Wrote this on Discord but you probably missed it. Regarding the Alchemy shard: I think it would maybe be a fun idea to make it a "brewery" shard? Wouldnt need a change of the art. Then you could do things like Potion Brewers, Alchemical Imbuers, a IE Fermenter Kit and a Mixer Kit, and have it require a lot of some liquids like Ethanol and Potions. I think thats more fun than the bottled versions. Maybe keep a few of the original ones like the bandages so that the original purpose doesnt get completely lost and you still have some potion bottles in there. I think the traps are fine aswell. That way you could throw in some more Simple Farming Brewery as well, and i think it would be thematically a good counterpart to the meals

File: config/ftbquests/quests/chapters/hidden_quests.snbt [hunk 1]
```diff
@@ -192,5 +192,14 @@
 				stage: "lesser_tartaric_gem"
 			}]
 		}
+		{
+			x: -14.0d
+			y: -6.5d
+			id: "2ECD3FE62B90722E"
+			tasks: [{
+				id: "354F1CF4ADCBD54A"
+				type: "checkmark"
```
Comment (NielsPilgaard) [inline config/ftbquests/quests/chapters/hidden_quests.snbt:201]: A checkmark task in a hidden quest?

File: kubejs/client_scripts/item_modifiers/jei_descriptions.js [hunk 2]
```diff
@@ -551,9 +551,51 @@ onEvent('jei.information', (event) => {
...
+        { type: 'whiskey', ingredient: 'Wheat', effect: 'Absorption' },
+        { type: 'wine', ingredient: 'Grapes', effect: 'Regeneration' }
+    ];
+
+    simplefarming_beverages.forEach((beverage) => {
+        recipes.push({
+            items: [`simplefarming:${beverage.type}`],
+            text: [
+                `Brewed by right-clicking ${beverage.ingredient} into a Brewing Barrel.`,
+                `Grants the imbiber with ${beverage.effect}.`
```
Comment (NielsPilgaard) [inline kubejs/client_scripts/item_modifiers/jei_descriptions.js:594]: The `with` seems missplaced - I think `Grants the imbiber ${beverage.effect}.` would be better.

File: config/ftbquests/quests/chapters/hidden_quests.snbt (hunk 1 above)
Comment (MuteTiefling) [reply config/ftbquests/quests/chapters/hidden_quests.snbt:201]: oh woops, that was a junk quest I was using to get NBT data

File: kubejs/client_scripts/item_modifiers/jei_descriptions.js (hunk 2 above)
Comment (MuteTiefling) [reply kubejs/client_scripts/item_modifiers/jei_descriptions.js]: Sounds good. Changed.

**Review by NielsPilgaard — APPROVED** [review]

**Comment by NielsPilgaard** [comment]
Would you like this PR merged now, or wait on the last 2 Shards?

File: kubejs/startup_scripts/item_registry.js [hunk 3]
```diff
@@ -351,99 +365,99 @@ onEvent('item.registry', (event) => {
...
+        { name: 'icy', band: 0x92b9fe, outline: 0x92b9fe },
+        { name: 'obsidian', band: 0x663399, outline: 0x663399 },
+        { name: 'pigman', band: 0x885956, outline: 0x885956 },
+        { name: 'rgbee', band: 0xffffff, outline: 0xffffff },
+        { name: 'rocky', band: 0x8f8f8f, outline: 0x8f8f8f },
+        { name: 'sand', band: 0xe7e4bb, outline: 0xe7e4bb },
+        { name: 'skeleton', band: 0xf6f2e6, outline: 0xf6f2e6 },
+        { name: 'slimy', band: 0x73c262, outline: 0x73c262 },
+        { name: 'water', band: 0x5979ef, outline: 0x5979ef },
+        { name: 'zombie', band: 0x2f4e32, outline: 0x2f4e32 },
```
Comment (Quezler) [inline kubejs/startup_scripts/item_registry.js:437]: monospace murderer 👀

Comment (NielsPilgaard) [reply kubejs/startup_scripts/item_registry.js:437]: Blame Prettier :P

Comment (MuteTiefling) [reply kubejs/startup_scripts/item_registry.js:437]: More like _thank_ Prettier

Comment (NielsPilgaard) [reply kubejs/startup_scripts/item_registry.js:437]: I like the monospace, but I like identical formatting everywhere more :D

Comment (Quezler) [reply kubejs/startup_scripts/item_registry.js:437]: felt neat to have the hex codes of the bees line up tho, but excluding just this block would require a comment 😅

**Pull request merged**